### PR TITLE
Basic docs for numpydoc

### LINKED ISSUE #66: numpydoc isn't really documented.
It'd be nice to move the numpydoc documentation outside the numpy repository and have a proper website on read the doc.
That should be quite easy to do and if of interest to the numpydoc community, be assigned to a URAP student here at BIDS? (@stefanv, what do you think?)

### PR DESCRIPTION
**Pull request by jnothman**
Apologies if my conf.py is naively constructed from a mix of the Sphinx generated and numpy's.

Fixes #66?

### CODE REVIEW

**Comment by pv** [comment]
Add it to travis build?

**Comment by jnothman** [comment]
If there is consensus that it's worth hosting numpydocs separate from numpy, how do you want to handle publishing?

**Comment by pv** [comment]
I think readthedocs is the minimum effort solution, automatic doc builds etc. without hassle.
(The builds on travis would only be for testing the doc build stays in working condition.)

**Comment by jnothman** [comment]
It's also the subject of #66

**Comment by jnothman** [comment]
So you just want an exit status check in travis? Sure.

File: doc/example.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+======
+
+.. literalinclude:: example.py
+   :caption: `example.py <http://github.com/numpy/numpydoc/blob/master/doc/example.py>`_
+   :language: python
+
+Rendered
+========
+
+.. automodule:: example
```
Comment (pv) [inline doc/example.rst]: Doesn't seem to import this module on travis, maybe add to sys.path in config?

**Comment by pv** [comment]
Also, the configuration options are probably best explained in just one place, so maybe drop from README.rst?

otherwise lgtm

**Comment by pv** [comment]
needs rebase

**Comment by pv** [comment]
lgtm, I'll just merge and if there's something to do we can iterate.
I created the readthedocs project --- let me know your account name to get admin rights there.

**Comment by jnothman** [comment]
Great! I'm jnothman there too.

**Pull request merged**